tests/templates/kuttl/external-access/opensearch.yaml.j2

```diff
@@ -11,7 +11,6 @@ spec:
     productVersion: "{{ test_scenario['values']['opensearch'].split(',')[0] }}"
     pullPolicy: IfNotPresent
 {% if lookup('env', 'VECTOR_AGGREGATOR') %}
-  clusterConfig:
     vectorAggregatorConfigMapName: vector-aggregator-discovery
```

Comment:
`clusterConfig` cannot be removed because `vectorAggregatorConfigMapName` is still there.